src/ds/ds-widget.js

```diff
@@ -136,7 +136,7 @@ jQuery(function ($) {
                             displayResults();
 
                             window.onscroll = function(ev) {
-                                if ($(window).scrollTop() + $(window).height() === $(document).height()) {
+                                if ($(window).scrollTop() + $(window).height() > 0.75 * $(document).height()) {
```

Comment:
This has nothing to do with the accessibility fixes in this PR. This is just a simple update to "infinite scrolling", so that the results load before user scrolls all the way to the bottom of the page. I didn't think we needed a whole new PR for just 7 extra characters.